index.js

```diff
@@ -18,9 +20,15 @@ app.use(express.json());
 // To parse the cookies when the refresh endpoint is hit
 app.use(cookieParser());
 
-//Middleware to log request to before it's handled (i.e. sent to the actual endpoint) 
+// Middleware to log request to before it's handled (i.e. sent to the actual endpoint) 
 app.use('/', logRequest);
 
+// Middleware to validate employee data before adding employee
+app.use('/employees/add', validateAddEmployeeData);
```

Comment:
Is it possible to add "validateAddEmployeeData" to the employeeRoutes.js file (similar to how you added `verifyToken` to the /add endpoint)?